### ISSUE TITLE
[Bug]: Record download fail

### ISSUE THREAD

**Issue by bootsmcleod** [Symbiota/community-board#562]
### Contact Details

daniel.paul.donovan@gmail.com

### Symbiota portal name

CCH2

### What happened?

I did a map search resulting in about 1,000 records. I tried to download a csv file, and it spins endlessly and never works. Same thing with a kml file. 

![image](https://github.com/BioKIC/symbiota-docs/assets/174574558/13ce86df-c90c-4008-8e25-af4aa87c0276)

### What operational systems are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

**Comment by themerekat** [comment]
Thanks for reporting, @bootsmcleod ! Do you remember what your search terms were?

**Comment by bootsmcleod** [comment]
Hi, Katie!

Yes, it was a small area and I wanted all records, so the search terms were
just United States, California. I think there were 1,004 records. I should
add that after i posted, a search for United States, California, 01/01/2017
to present, yielding about 825 records, did download in about 2 seconds.

Thanks,
Daniel

**Comment by themerekat** [comment]
@bootsmcleod , aha, so you didn't have a polygon or bounding box included in your search terms, though?
If that's the case, then the search was being conducted over _all_ United States, California records, which would probably be over 3 million records. The download tools only allow 1 million at a time, so the download may have stalled.

**Comment by bootsmcleod** [comment]
No, there was a bounding box of about 19 sq km. That's why only 1,004
records

**Comment by themerekat** [comment]
Oh ok, thanks! Can you try a bounding box search somewhere else and see if you're having the same problem today?

**Comment by bootsmcleod** [comment]
Hi, Katie!

Sorry for the delay. I did a similar search in a different place on the
map, and the CSV still won't download. I'm also on a different computer.
The latest search was a polygon on San Mateo Canyon in the Santa Ana Mtns,
search terms: California, 01/01/2000-07/08/2024, all vascular plant
collections. The search yielded 565 records.

On another note, I've noticed that if several records are georeferenced to
the same point, now only one point shows on the map. The others will appear
in the list, but hovering over the point no longer gives you all the
records that are mapped there, just one.

[image: image.png]

Thanks!
Daniel

**Comment by themerekat** [comment]
Thanks @bootsmcleod ! We're still working on this issue.
Regarding your second issue, we just approved a fix for this, too. Should be up in the next few days.
